src/Components/Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../api";
+import { useParams } from "react-router-dom";
+import { ListGroup } from "react-bootstrap";
+import { formatDate } from "../utils";
+function Comments() {
+    const [comments, setComments] = useState([]);
+    const { article_id } = useParams();
+  
+    useEffect(() => {
+      getComments(article_id)
```

Comment:
If you were a user would you want to see a loading state while comments are loading?